src/squad.cpp

```diff
@@ -129,7 +129,7 @@ void Squad::read_members() {
 
         m_uniforms.insert(position,u);
         LOGD << "checking orders for position" << position;
-        foreach(VIRTADDR ord_addr, m_df->enumerate_vector(addr+0x4)){ //TODO: offset
+        foreach(VIRTADDR ord_addr, m_df->enumerate_vector(addr+0x2)){ //TODO: offset

```

Comment:
Is this something that should be correct in the layouts instead?

Reply:
Changing this back to addr+0x4 fixes the squad crash on startup for me.